source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -2236,6 +2236,28 @@ bool COpenGLDriver::SAuxContext::setActiveTexture(uint32_t stage, core::smart_re
 	return true;
 }
 
+void COpenGLDriver::SAuxContext::setActiveBindings(const COpenGLDescriptorSet* descriptorSets[4])
```

Comment:
nope, not all at once, do a 
`setActiveDescriptorSet(uint32_t index, const COpenGLDescriptorSet* descSet)`

Reply:
@Crisspl look at the vulkan command buffer operation

Reply:
@Crisspl where is this function now?